js/components/collapsible.js

```diff
@@ -0,0 +1,48 @@
+// Walkhub
+// Copyright (C) 2015 Pronovix
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import React from "react";
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
+
+class Collapsible extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {isExpanded: this.props.isExpanded};
+    this.clickTitle = this.clickTitle.bind(this);
+  }
+  clickTitle() {
```

Comment:
this could be `clickTitle = () => {`, so the reassign/bind won't be necessary in the constructor.